### DISCUSSION TITLE
Adding a dummy atom to the PQR file without interfering the calculation

### DISCUSSION THREAD

**Discussion by rwxayheee** [Electrostatics/apbs#186] [Q&A]
Dear APBS experts, 

I'm using APBS version 3.2.1 on Linux x86_64 OS, and I was using the "mg-auto" mode to calculate the electrostatic potential map of a protein. 

And I need to add a dummy atom (with no radius and no charge) to the PQR file. The dummy atom is added at the end of the PQR file: 

> (protein atoms)
> ATOM   8259  OXT THR   543      63.532 103.325 -14.812 -0.8044 1.6612
> ATOM   8260  HG1 THR   543      67.879 103.234 -16.564  0.4119 0.0000
> TER
> ATOM   9999  UNK FOO   999      91.488  86.764  -3.189  0.0    0.0
> END

From the standard output of APBS calculation, I checked the "Total electrostatic energy" value at the last step "CALCULATION #3: MULTIGRID" for the fine mesh region. There seems to be a minor difference (~0.5 kJ/mol) caused by adding the dummy atom: same calculation without the dummy atom gives a total electrostatic energy of 2.384663622213E+05 kJ/mol, and the calculation with the dummy atom gives a total electrostatic energy of 2.384669050542E+05 kJ/mo. 

Could you please tell me what is the correct syntax to completely exclude the dummy atom from the calculation? 

Thank you for your kind advice in advance. 

Bests,
Amy

**Comment by sobolevnrm** [comment]
Hello -- what dielectric surface definition (`srfm`) are you using?

**Reply by rwxayheee** [reply]
Hi Nathan, Thank you for your kind reply. I'm using "smol". My input file is as follows: 

> read
>     mol pqr protein.pqr
> end
> elec
>     mg-auto
>     dime 289 289 289
> 
>     cglen 1000 1000 1000
>     fglen 144 144 144
> 
>     cgcent mol 1
>     fgcent mol 1
> 
>     mol 1
>     lpbe
>     bcfl sdh
> 
>     ion 1 0.1 1.62 # Na+ Born radius, doi/10.1021/jp509445k
>     ion -1 0.1 2.26 # Cl- Born radius, doi/10.1021/jp509445k
> 
>     pdie 4
>     sdie 78
>     srfm smol
> 
>     chgm spl2
> 
>     sdens 10.0
>     srad 1.40
>     swin 0.30
>     temp 298
>     calcenergy total
> 
>     calcforce no
>     write pot dx protein
> end
> quit

**Comment by sobolevnrm** [comment]
Hello --

`srfm mol` and `srfm smol` will generate surface contributions from zero-radius atoms.  They probably shouldn't do that but that's a low-priority bug to fix.

What are you trying to accomplish with the dummy atom?  There may be other ways to get the information you're looking for.

Thank you

**Reply by rwxayheee** [reply]
Hi Nathan, 
Thank you for your kind reply. I'm doing Brownian Dynamics simulations in program [Browndye](https://browndye.ucsd.edu/), and I was trying to define the reaction surface by a sphere centered at the geometric center of a few atoms. Since the dummy atom might be taken into account in APBS calculation, I will try to defined the surface by a combination of actual atoms instead. Thank you again for the information! 

Thanks,
Amy
